package.json

```diff
@@ -15,6 +16,7 @@
     "react-router-dom": "^6.2.2",
     "react-scripts": "5.0.0",
     "redux": "^4.1.2",
+    "redux-logger": "^3.0.6",
```

Comment:
- [ ] Please, kindly implement the [Microverse Readme Template](https://github.com/microverseinc/readme-template), instead of the CRA getting started readme.  All sections are required, except the "Tests" and "Live demo" if not required by the project.

That way, your project will be well documented, and we will meet this general requirement 💪

Reply:
Updated! Thank you for the suggestion.